package.json

```diff
@@ -44,6 +44,6 @@
     "build": "babel src --out-dir .",
     "check": "node_modules/.bin/eslint --ext \".js\" --ignore-pattern node_modules --quiet .",
     "test": "NODE_ENV=test mocha --opts test/mocha.opts",
-    "postinstall": "rm -f .babelrc && npm run build"
+    "postinstall": "npm run build"
```

Comment:
nothing to do here, lets go ahead and I'll test if this doesn't break anything in RN